Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -303,15 +303,12 @@ - (DisplayableText *)displayableTextForInteractionId:(NSString *)interactionId
     if (!displayableText) {
         NSString *text = textBlock();
 
-        // Only show up to 2kb of text.
-        const NSUInteger kMaxTextDisplayLength = 2 * 1024;
-        text = [text ows_stripped];
+        // Only show up to N characters of text.
+        const NSUInteger kMaxTextDisplayLength = 1024;
```

Comment:
I propose we should less by default.  WDYT?

Reply:
:+1: